### PR TITLE
Change GS Nation to ReaperScans.fr

### PR DESCRIPTION
**Pull request by LeMyst**
Use Reaper Scans US icon

### CODE REVIEW

**Comment by LeMyst** [comment]
I wait for correct extension name from scantrad team.

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt [hunk 1]
```diff
@@ -29,7 +29,7 @@ class WPMangaReaderGenerator : ThemeSourceGenerator {
         SingleLang("Skull Scans", "https://www.skullscans.com", "en"),
         SingleLang("Luminous Scans", "https://www.luminousscans.com", "en"),
         SingleLang("Azure Scans", "https://azuremanga.com", "en"),
-        SingleLang("GS Nation", "https://gs-nation.fr", "fr", overrideVersionCode = 1),
+        SingleLang("ReaperScans.fr", "https://reaperscans.fr/", "fr", className  = "ReaperScansFR", pkgName = "gsnation", overrideVersionCode = 2),
```
Comment (jobobby04) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt:32]:
```suggestion
        SingleLang("ReaperScans.fr", "https://reaperscans.fr", "fr", className  = "ReaperScansFR", pkgName = "gsnation", overrideVersionCode = 2),
```
Having that last slash can break the source

File: multisrc/overrides/wpmangareader/gsnation/src/ReaperScansFR.kt [hunk 2]
```diff
@@ -8,7 +8,10 @@ import org.jsoup.nodes.Document
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class GSNation : WPMangaReader("GS Nation", "http://gs-nation.fr", "fr", dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale.FRANCE)) {
+class ReaperScansFR : WPMangaReader("ReaperScans.fr", "https://reaperscans.fr/", "fr", dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale.FRANCE)) {
```
Comment (jobobby04) [inline multisrc/overrides/wpmangareader/gsnation/src/ReaperScansFR.kt:11]:
```suggestion
class ReaperScansFR : WPMangaReader("ReaperScans.fr", "https://reaperscans.fr", "fr", dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale.FRANCE)) {
```

File: multisrc/overrides/wpmangareader/gsnation/src/ReaperScansFR.kt [hunk 3]
```diff
@@ -8,7 +8,10 @@ import org.jsoup.nodes.Document
...
+
+    // Formerly "GS Nation"
+    override val id = "8953394032396297337"
```
Comment (alessandrojean) [inline multisrc/overrides/wpmangareader/gsnation/src/ReaperScansFR.kt:14]: `id` is a Long, you need to remove the quotes.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

File: multisrc/overrides/wpmangareader/gsnation/src/ReaperScansFR.kt (hunk 3 above)
Comment (LeMyst) [reply multisrc/overrides/wpmangareader/gsnation/src/ReaperScansFR.kt:14]: I changed it in the generated source but not in the origine. :(
Will fix

**Review by alessandrojean — CHANGES_REQUESTED** [review]

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt [hunk 4]
```diff
@@ -29,7 +29,7 @@ class WPMangaReaderGenerator : ThemeSourceGenerator {
         SingleLang("Skull Scans", "https://www.skullscans.com", "en"),
         SingleLang("Luminous Scans", "https://www.luminousscans.com", "en"),
         SingleLang("Azure Scans", "https://azuremanga.com", "en"),
-        SingleLang("GS Nation", "https://gs-nation.fr", "fr", overrideVersionCode = 1),
+        SingleLang("ReaperScans.fr", "https://reaperscans.fr", "fr", className  = "ReaperScansFR", pkgName = "gsnation", overrideVersionCode = 2),
```
[the remark this answers is not in the document]
Comment (LeMyst) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt:32]: I already bumped version from 1 to 2.

**Comment by LeMyst** [comment]
Going for "ReaperScans.fr (GS)" as requested by the team

**Pull request merged**